tests/cypress/support/commands/global.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/no-namespace */
```

Comment:
Please, never disable eslint rules for the whole file.
When needed, always disable it locally for specific line or codeblock.

You can put it around codeblock like:
```
/* eslint-disable some/rule */
   some codeblock here, for example a function or interface...
/* eslint-enable some/rule */